packages/components/src/components/forms/input/AliasInput.tsx

```diff
@@ -5,25 +5,44 @@ import { generateId, QueryColumn, SelectInput } from '../../..';
 interface AliasInputProps {
     col: QueryColumn;
     editing?: boolean;
-    value?: string;
+    value?: string | Array<Record<string, any>>;
     allowDisable?: boolean;
+    initiallyDisabled: boolean;
+    onToggleDisable?: (boolean) => void;
 }
 
-export class AliasInput extends React.Component<AliasInputProps, any> {
+interface AliasInputState {
+    labelKey?: string;
+}
+
+export class AliasInput extends React.Component<AliasInputProps, AliasInputState> {
     _id: string;
 
     constructor(props: AliasInputProps) {
         super(props);
 
         this._id = generateId();
+
+        // Alias fields could use label or displayValue
+        let labelKey = 'label';
+        if (typeof props.value !== 'string') {
+            if (props.value[0] && 'displayValue' in props.value[0]) {
```

Comment:
if props.value is undefined this produces an error.

Reply:
Yes this was a bug. Fixed with better type and length checking